src/content/pages/attend/travel.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Travel Tips & Tricks
+order: 10
+---
+
+
+# Travelling to PyCon AU 2024
+
+For some folks, PyCon AU 2024 will be their first time in Melbourne. Welcome! For others, it might have been a few years so welcome back! Here are some advice for getting around town (and to the conference).
+
+## Interstate & International travel
+Book travel to Melbourne Tullamarine Airport (MEL). Melbourne also has a second airport, Avalon Airport (AVV), which is significantly further from the city and less well-served by public transport.
+
+Both MEL and AVV are served by [SkyBus](https://www.skybus.com.au/), a shuttle bus service that runs regularly between airports and the city. Tickets can be purchased upon arrival but are cheaper ($40 return for an adult) if pre-booked online.
+
+## Public transport
+Melbourne has a single ticketing system for all public transport, including trains, trams, and buses. The system is called [Myki](https://www.ptv.vic.gov.au/tickets/Myki). You can purchase a Myki card at the airport, most train stations, and many convenience stores around town. The card costs $6 and needs to be topped up with money for travel. 
+
+### Trams
+Melbourne has an extensive and easy to use tram network. Travel is free inside [the green free tramzone](https://www.ptv.vic.gov.au/assets/PDFs/Maps/Network-maps/5b4a3efe80/PTV-Free-Tram-Zone-Map.pdf) covering most of the CBD, but you need to 'tap on' with your Myki if travelling outside hte green zone.
```

Comment:
`tramzone` should probs be `tram-zone`

`hte` to `the

Reply:
Also worth mentioning the venue itself is not within the FTZ